src/widgets/header/ui/subaccount-selector.tsx

```diff
@@ -25,7 +25,7 @@ export const SubaccountSelector = observer(() => {
         <DropdownMenu.Trigger>
           <Button>
             {isLoading ? (
-              <div className='min-w-[170px] flex justify-center'>
+              <div className='min-w-[142px] flex justify-center'>
```

Comment:
Loading indicator wasn't quite the right size to reduce the screenjank

Reply:
I might have old man eyes, because I can't see this on my computer 😅 . Maybe we can revisit this later.